paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt

```diff
@@ -118,24 +117,20 @@ internal class PaymentOptionsViewModelTest {
         }
 
     @Test
-    fun `resolveTransitionTarget no new card`() {
+    fun `Opens saved payment methods if no new payment method was previously selected`() = runTest {
         val viewModel = createViewModel(
             args = PAYMENT_OPTION_CONTRACT_ARGS.updateState(newPaymentSelection = null)
         )
 
-        var transitionTarget: TransitionTarget? = null
-        viewModel.transition.observeEventsForever {
-            transitionTarget = it
+        viewModel.currentScreen.test {
+            skipItems(1)
```

Comment:
Would it be better to assert this one is `null`, with a comment saying we're still loading?

Reply:
I can assert it as `null` and then update the test when adding the `TransitionTarget.Loading`. Does that work?